apps/coscrad-frontend/src/components/resources/shared/resource-preview.tsx

```diff
@@ -0,0 +1,16 @@
+import { AggregateType } from '@coscrad/api-interfaces';
+
+interface ResourcePreviewProps {
+    resourceType: AggregateType;
+}
+
+const mediaUrl = 'https://kaaltsidakah.net/coscrad/resource-icons';
+const iconDimension = '80px';
+
+export const ResourcePreview = (props: ResourcePreviewProps): JSX.Element => (
```

Comment:
Rename this. Consider
- ResourceIcon
- ResourcePreviewIcon (if there will be no other use cases)